### DISCUSSION TITLE
ignore templates folder in Obsidian_to_Anki

### DISCUSSION THREAD

**Discussion by ramseylove** [BryanJenksCommunity/FAQ#148] [Q&A] [answered]
After adding templates for each of my Anki card types. I noticed that all my newly created templates were added to a new "blank" deck in Anki. It obviously makes sense why they were added but i'm wondering if there is a way to have the Obsidian_to_Anki plugin ignore the Templates directory, so they do not clutter up my Anki?
Thanks!

**Comment by tallguyjenks** [comment, marked as answer]
i ran into this same problem but its been a while, if i remember correctly i believe the solution was something like do the import and let them hit the default deck, but then suspend those cards and don't delete them, remove the ID tags from the templates and go on with life? but i could be wrong its been a while since i used the feature
